anesthetic/plot.py

```diff
@@ -120,6 +120,11 @@ def make_2d_axes(params, **kwargs):
             Figure to plot on.
             Default: matplotlib.pyplot.figure()
 
+        inner_ticks: bool
+            If False, plot ticks only on the very left and very bottom.
+            If True, plot ticks also in inner subplots.
+            Default: False
```

Comment:
I first had this set up on my local branch to permanently change the plot behaviour, because I quite generally preferred having ticks, especially for parameters which might have some meaningful values (Omega_K comes to mind). However, I found that it was actually quite easy to make it optional (note, it is generally _not_ that easy to make these ticks work in this kind of lower/diagonal/upper setup), so that's what I did for now. This does mean having an additional `inner_ticks` kwarg, though. Better name suggestion?

Do we like this optional kwarg? Or would we prefer fewer kwargs and picking one appearance? If we prefer no ticks then I'll close this and keep it in my local branch. If we prefer always ticks then I can remove the extra kwarg.